x/auth/client/cli/account.go

```diff
@@ -72,4 +55,5 @@ func GetAccountCmd(storeName string, cdc *codec.Codec, decoder auth.AccountDecod
 			return nil
 		},
 	}
+	return client.GetCommands(cmd)[0]
```

Comment:
this is a bit unintuitive - why do we not just return `cmd` ?

Reply:
Because we have to set some flags here. Setting the flags where the command is defined seems to make the most sense.

Reply:
ok, maybe add a comment but nbd

Reply:
Added!